# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/parent/FileSystemManagerParent.cpp
# Commit: 399dce373c59
# Full Hash: 399dce373c5917ac141f7fd341ed7c4dca56144a
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 3
# Description:
#   Bug 1786465 - Move FileSystemDataManager to own source files; r=dom-storage-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D155550
# ==============================================================================

diff -r f62bd47cabd3 -r 399dce373c59 dom/fs/parent/FileSystemManagerParent.cpp
--- a/dom/fs/parent/FileSystemManagerParent.cpp	Wed Aug 31 09:34:53 2022 +0000
+++ b/dom/fs/parent/FileSystemManagerParent.cpp	Wed Aug 31 09:34:54 2022 +0000
@@ -6,6 +6,7 @@
 
 #include "FileSystemManagerParent.h"
 #include "nsNetCID.h"
+#include "mozilla/dom/FileSystemDataManager.h"
 #include "mozilla/dom/FileSystemTypes.h"
 #include "mozilla/ipc/Endpoint.h"
 
@@ -24,7 +25,7 @@
 
 FileSystemManagerParent::FileSystemManagerParent(TaskQueue* aTaskQueue,
                                                  const EntryId& aRootEntry)
-    : mTaskQueue(aTaskQueue), mData(), mRootEntry(aRootEntry) {}
+    : mTaskQueue(aTaskQueue), mRootEntry(aRootEntry) {}
 
 IPCResult FileSystemManagerParent::RecvGetRootHandleMsg(
     GetRootHandleMsgResolver&& aResolver) {